components/server/src/workspace/garbage-collector.ts

```diff
@@ -43,8 +43,9 @@ export class WorkspaceGarbageCollector {
     }
 
     public async garbageCollectWorkspacesIfLeader() {
+        const initialLockDuration = 5 * 60 * 1000; // 5 minutes
         try {
-            await this.mutex.client().using(["workspace-gc"], 30 * 1000, async (signal) => {
+            await this.mutex.client().using(["workspace-gc"], initialLockDuration, async (signal) => {
```

Comment:
The alternative reasonable value would be the actual frequency at which we plan to run the job. The lock is auto-released once the function completes, so it shouldn't be an issue